### DISCUSSION TITLE
my attending campus doesn't exist on the dropdown of schools

### DISCUSSION THREAD

**Discussion by b4nbird** [community/community#52256] [GitHub Education]
I am a student of Nanjing University of Science and Technology，but my school has two campuses in different places，and my attending campus doesn't exist on the dropdown of schools，so my application failed due to geographic location
![图片](https://user-images.githubusercontent.com/63333153/230610264-ebae6f49-83c9-48e9-b4a2-ee2bca76a7ad.png)
the campus's webisite is https://jiangyin.njust.edu.cn/，and as you can see，it Indicated in the official website of Nanjing University of Science and Technology
![图片](https://user-images.githubusercontent.com/63333153/230611500-0ad2822d-1a9c-46d9-be8a-dabe3f395db4.png)
please tell me the solution if i want to apply the GitHub Global Campus

**Comment by samus-aran** [comment]
Hi @b4nbird 

We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.

Here is a quick tip on submitting evidence:

- If your original document is not in English then we advise you to take a photo of the original next to an English translation. This also include your profile information.
- Ensure your evidence correspond to details in your GitHub profile. The verification process looks at your GitHub username, email and name ( as well as other points) to cross reference the evidence submitted. Do you use a variation of your name or a nickname?  It’s these small but important points to consider when applying that can make a much smoother process.
- In order to qualify for the GitHub education program, we must be able to verify that you are either located close to your campus, or that you are a part of a verifiable distance learning program.

You will have the most success applying if you are able to do so from your school’s campus.

- When asked, please enable the application to discover your location from the browser pop-up. Since settings for every browser are different, if you are having difficulty enabling us to see your location you should seek assistance from your browser's support pages.
- If you are using a VPN, please disable it when you reapply. 
- If you are not using a VPN, try reapplying when connected to the internet from a different network.

Let us know how you get on after applying these above tips 👍
